docs/execution-analysis/debug.md

```diff
@@ -300,17 +275,17 @@ You can seek help with your issue by asking a question directly in the [communit
 
 ## Submitting an issue
 
-If you cannot find a solution in the community forum, or if you have found a bug in the library, you could create an issue in our GitHub repository.
+If you cannot find a solution in the community forum, or if you have found a bug in the library, you could [create an issue](https://github.com/zama-ai/concrete/issues/new/choose) in our GitHub repository.
 
-In case of a bug, try to:
+For [bug reports](https://github.com/zama-ai/concrete/issues/new?assignees=&labels=bug%2C+triage&projects=&template=bug_report.md), try to:
 
-* minimize randomness;
-* minimize your function as much as possible while keeping the bug - this will help to fix the bug faster;
-* include your inputset in the issue;
-* include reproduction steps in the issue;
-* include debug artifacts in the issue.
+* Minimize randomness
+* Simplify your function while keeping the bug to expedite the fix
```

Comment:
I think "Minimize" is the right verb, it's commonly used and more precise that just "Simplify"